torch/csrc/jit/mobile/backport_manager.cpp

```diff
@@ -212,6 +213,35 @@ bool backport_v5_to_v4(
   return true;
 }
 
+bool backport_v6_to_v5(
+    std::stringstream& input_model_stream,
+    std::stringstream& output_model_stream) {
+  std::shared_ptr<IStreamAdapter> rai =
+      std::make_shared<IStreamAdapter>(&input_model_stream);
+  auto reader = std::make_shared<PyTorchStreamReader>(rai);
```

Comment:
Actually, since reader is not used anywhere else, we probably don't a `share_ptr`:
```
auto reader = PyTorchStreamReader(&input_model_stream);
```
Accordingly, change `reader->hasRecord` to `reader.hasRecord`.